test-clients/src/main/resource/contract/solidity/ZenosBank.sol

```diff
@@ -17,17 +19,21 @@ contract ZenosBank {
     }
 
     function depositTokens(int64 amount) public {
-        int response = HederaTokenService.transferToken(tokenAddress, msg.sender, address(this), amount);
-        if (response == 22) {//FIXME HederaResponseCodes.SUCCESS) {
+        (bool success, bytes memory result) = precompileAddress.delegatecall(
```

Comment:
We should be using the `HederaTokenService` functions instead of doing the direct calls with `delegateCall`.